packages/swingset-xsnap-supervisor/src/index.js

```diff
@@ -0,0 +1,23 @@
+import fs from 'fs';
+import { bundlePaths } from './paths.js';
+
+export { supervisorBundleSHA256 } from '../dist/supervisor.bundle.sha256.js';
+
+/**
+ *
+ * @typedef {{ moduleFormat: string, source: string, sourceMap: [string] }} Bundle
+ * @returns { Promise<Bundle> }
+ */
+export const getSupervisorBundle = async () => {
+  const path = bundlePaths.supervisor;
+  const bundleStringP = fs.promises.readFile(path, { encoding: 'utf-8' });
+  return bundleStringP
+    .catch(err => {
+      console.error(`supervisor bundle not present at ${path}`);
```

Comment:
Nit: technically it's only not present if `err.code === 'ENOENT'`

Reply:
I'll soften the error message, "cannot read..", the extra error message is just to give a hint to the user of the most common reason for failure